<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParametersMRVM</class>
 <widget class="QWidget" name="ParametersMRVM">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>310</width>
    <height>144</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="labelWidth">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>10</y>
     <width>80</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Kernel Width</string>
   </property>
  </widget>
  <widget class="QLabel" name="svmTypeLabel">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>C</string>
   </property>
  </widget>
  <widget class="QComboBox" name="svmTypeCombo">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>Type of SVM training:
C-SVM (or eps-SVM): C cost based SVM, trained via SMO
Nu-SVM: Nu based SVM, trained via SMO
RVM: Relevant Vector Machine
Pegasos: Online trainer for SVM</string>
   </property>
   <item>
    <property name="text">
     <string>C-SVM</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>nu-SVM</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Pegasos</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Type</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="kernelWidthSpin">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>30</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>Width of the kernel (gamma)
RBF and Polynomial only</string>
   </property>
   <property name="decimals">
    <number>3</number>
   </property>
   <property name="minimum">
    <double>0.001000000000000</double>
   </property>
   <property name="maximum">
    <double>9999.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.010000000000000</double>
   </property>
   <property name="value">
    <double>0.100000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="svmCSpin">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>30</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>Training Parameter
C-SVM: C cost function, (alpha_i &lt;= C)
Nu-SVM: nu, ratio on the amount of alphas that can be selected
RVM: eps, stopping criterion (the smaller, the more accurate)
Pegasos: lambda, accuracy-generalization tradeoff parameter </string>
   </property>
   <property name="decimals">
    <number>3</number>
   </property>
   <property name="minimum">
    <double>0.001000000000000</double>
   </property>
   <property name="maximum">
    <double>999.990000000000009</double>
   </property>
   <property name="singleStep">
    <double>0.010000000000000</double>
   </property>
   <property name="value">
    <double>1.000000000000000</double>
   </property>
  </widget>
  <widget class="QCheckBox" name="optimizeCheck">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>60</y>
     <width>110</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Optimize Kernel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
